xpdacq/beamtime.py

```diff
@@ -308,7 +308,7 @@ def _plan_validator(self):
                 scan tyoe of XPD Scan object
         '''
         # based on structures in xpdacq.xpdacq.py
-        _Tramp_required_params = ['startingT', 'endingT', 'Tstep', 'exposure']
+        _Tramp_required_params = ['startingT', 'endingT', 'requested_Tstep', 'exposure']
```

Comment:
I changed this to ``Tstep``.  ``requested_Tstep`` is too much to ask our users to type, and it is redundant since it is obviously "requested" since they are typing (and therefore "requesting") it.   Later on we will call it "requested" internally to differentiate it from the actual one that is computed to be as close to, but not necessarily the same as, the requested one.  However, we should not expose our users to this.  

What it means is that it should be called ``Tstep`` when it is in a dictionary or method argument that is typed by a user, but saved in metadata as ``requested_Tstep``